src/services/pages.js

```diff
@@ -0,0 +1,33 @@
+import generateApiUri from "../components/utils/generateURI";
+
+const getPages = async (fileKey, { beforeVersion, afterVersion }) => {
+  const baseURI = import.meta.env.VITE_BACKEND_BASE_API_URI;
+
+  const token = JSON.parse(localStorage.getItem("FigmaToken")).access_token;
+
+  const queryParams = {
+    "before-version": beforeVersion,
+    "after-version": afterVersion,
+  };
+
+  const API_URI = generateApiUri(
+    baseURI,
+    `projects/${fileKey}/pages`,
+    queryParams,
+  );
+
+  const response = await fetch(API_URI, {
+    method: "GET",
+    headers: {
+      authorization: token,
+    },
+  });
+
+  const responseJson = await response.json();
+
+  return responseJson;
+};
+
+const getPageDiff = () => {};
+
+export { getPages, getPageDiff };
```

Comment:
- `P2`
서버에서는 `diffingResult`라는 변수명으로 보내고 있으므로

```suggestion
export { getPages, getDiffingResult };
```
로 통일하면 더 좋을 것 같습니다